src/bin/shim-sgx/build.rs

```diff
@@ -0,0 +1,10 @@
+// SPDX-License-Identifier: Apache-2.0
+
+fn main() {
+    println!(
+        "cargo:rustc-link-arg-bin=enarx-shim-sgx=-T{}/layout.ld",
+        env!("CARGO_MANIFEST_DIR")
```

Comment:
That means, that the user must not ever move the parent directories around... right?

Reply:
once build.rs is compiled